.github/workflows/tests-nightly.yml

```diff
@@ -67,21 +67,21 @@ jobs:
       run: |
         echo "Running unit tests for server: ${{ matrix.testServer }}"
         mkdir -p ~/logs-unit
-        ./gradlew runTestsWithTag -PtestTag=unit --scan -Pserver=${{ matrix.testServer }} -PcontextDir=$HOME/logs-unit
+        ./gradlew runTestsWithTag -PtestTag=unit -Pserver=${{ matrix.testServer }} -PcontextDir=$HOME/logs-unit
```

Comment:
sad that the non-enterprise version doesn't have scan anymore

Reply:
Can we keep the --scan? (Same for rest of PR)

Reply:
good point, we can continue using the public scan functionality!